### PR TITLE
Allow disabling printing alpaca image to console using PRINT_ALPACA config parameter

### PR DESCRIPTION
**Pull request by kolomenkin**
## Proposed Changes

1. Allow disabling printing alpaca image to console using PRINT_ALPACA config parameter.

I'm using mega linter manually from console. Printing too mush useless information frustrates.
Alpaca image is nice to get in logs of CI. Or when you run the tool once a day.
I'm running the tool too much.

### Author/Contributor
- [x] If documentation is needed for this change, has that been included in this pull request

### Reviewing Maintainer
- [ ] Label as `breaking` if this is a large fundamental change
- [`enhancement`] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

**Comment by nvuillam** [comment]
Thanks for the contribution :) The alpaca may be sad to disappear on demand, but I understand you could want to have smaller logs :)

Please can you:
- Just update README and CHANGELOG at the root level ( the ones in doc are automatically generated)
- Run build.sh ( you will see updated README and CHANGELOG in /docs folder)
- Add a test case using PRINT_ALPACA=false (i'm not sure, as env vars are strings... maybe "true" instead of True for tests ?

**Review by nvuillam — CHANGES_REQUESTED** [review]
please check comment

**Comment by kolomenkin** [comment]
I do not get why do you have two change logs. I updated both of them.

Feel free to request any changes.

I will fix merge conflicts between my PRs once one of them is merged. Don't worry.

**Comment by kolomenkin** [comment]
I just read your comment. I will update documentation.
I will review your code again regarding the default value for PRINT_ALPACA. I have tested the code only against boolean value from yaml configuration file.

**Comment by nvuillam** [comment]
> I do not get why do you have two change logs. I updated both of them.
> 
> Feel free to request any changes.
> 
> I will fix merge conflicts between my PRs once one of them is merged. Don't worry.

README and CHANGELOG at root repository are the ones available for GitHub repo view
All online documentation (mkdocs-material) is generated by build.by, and to be available they must be in /docs folder

File: requirements.dev.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+gitpython
```
Comment (nvuillam) [inline requirements.dev.txt:1]: I already have all those packages in megalinter/setup.py , do we need to duplicate them here ? ( if yes no problem, I just don't know)

**Comment by kolomenkin** [comment]
- fixed the code for reading config/env variable.
- fixed documentation. I have also run `build.sh` and commited changes.
- added `requirements.dev.txt`. It is a subject to discuss on its place and naming, But I feel I needed something like this to build local virtual env for contribution. Maybe there are some packets which are not needed. Please note.

**Comment by nvuillam** [comment]
> * added `requirements.dev.txt`. It is a subject to discuss on its place and naming, But I feel I needed something like this to build local virtual env for contribution. Maybe there are some packets which are not needed. Please note.
> 
> Maybe there was a easy way to reuse setup.py Please help. I usually do not change python packages

As it's not called by production package, it can not hurt, ok to let it :)
Once the CI is ok i merge :) 
- Mega-Linter: https://github.com/nvuillam/mega-linter/pull/116/checks?check_run_id=1481968945
- build & pytest: https://github.com/nvuillam/mega-linter/pull/116/checks?check_run_id=1481968977

**Comment by nvuillam** [comment]
Note: it seems GitHub actions have problems... let's be patient 

https://www.githubstatus.com/

**Comment by kolomenkin** [comment]
> Note: it seems GitHub actions have problems... let's be patient
> 
> https://www.githubstatus.com/

I see you pushed new commits to master. Should I rebase?

**Comment by kolomenkin** [comment]
> please check comment

sorryI have lost what code line you tried to comment here. And now github expects some code changes from me according to your comment.

**Comment by nvuillam** [comment]
> > Note: it seems GitHub actions have problems... let's be patient
> > https://www.githubstatus.com/
> 
> I see you pushed new commits to master. Should I rebase?

Don't bother, I just disabled a useless workflow job
As you are the first other contributor than me (I hope there will be much more soon, don't forget to star the repo ⭐ ) , some upgrades may still be needed to enhance contributor experience, don't hesitate to advise updates about that !

**Review by nvuillam — APPROVED** [review]
Thanks for your contribution ! 🥇

**Pull request merged**